tests/django_pg_migration_tools/test_operations.py

```diff
@@ -427,3 +454,355 @@ def test_when_not_allowed_to_migrate(self):
                 {"table_name": "example_app_charmodel", "index_name": "char_field_idx"},
             )
             assert cursor.fetchone()
+
```

Comment:
Should there be a test for the scenario where the index does not already exist initially for completeness' sake?

Reply:
Added here: 6298fed4e45251df2163a7499c38b0fc0920f854